.github/ISSUE_TEMPLATE/extension-request.md

```diff
@@ -4,7 +4,6 @@ about: 'Extension/feature request'
 title: 'Insert name of the extension/feature here'
 labels: '🙋 Extension/feature request'
 assignees: ''
-
```

Comment:
This space is present in all of the templates. It is unnecessary to edit this and it could break the templates, however, I'm not sure.

Reply:
I'll check if removing whitespaces on metadata of a template might make it broken on my fork first.

Reply:
> I'll check if removing whitespaces on metadata of a template might make it broken on my fork first.

![image](https://user-images.githubusercontent.com/34998342/96363364-685a0b00-1166-11eb-88f5-59362558e553.png)
@CWKevo Nothing to happened in my fork.

Reply:
I remember that I simply fixed the issue we had earlier by just copying the bugs report template that worked and edited it to match the extension request one. I also remember that whitespaces were not the issue here, so I don't know what might've caused it to not be recognized by GitHub.

Reply:
Maybe it related to the CRLF/LF stuff if you're using VS Code in Windows.